CONTRIBUTING.md

```diff
@@ -0,0 +1,289 @@
+
+    This will output a bytestring, e.g. `b'mystring='`. Copy only the contents
+    of `'...'`, and add it to your `.env` file as `DB_ENCRYPTION_KEY`, e.g.
+    `DB_ENCRYPTION_KEY="mystring="`.
+
+    To exit the Python shell, press `Ctrl-Z` and then `Enter` on Windows, or
+    `Ctrl-D` on OS X or Linux. Alternatively, you could also type the Python
+    command `exit()` and press `Enter`.
+
+    Finally, set the following environment variables (if you\'re an OddBird, you
+    can find these values in the shared Keybase team folder \--`metecho/env`):
+
+        DOCKER_SFDX_HUB_KEY=...
+        SFDX_CLIENT_ID=...
+        SFDX_CLIENT_SECRET=...
+        GITHUB_HOOK_SECRET=...
+        GITHUB_CLIENT_ID=...
+        GITHUB_CLIENT_SECRET=...
+        GITHUB_APP_ID=...
+        DOCKER_GITHUB_APP_KEY=...
+
+    Note that none of the values should be quoted, and while
+    `DOCKER_SFDX_HUB_KEY` and `DOCKER_GITHUB_APP_KEY` are RSA private keys, they
+    must have newlines replaced with `\n` in order to work properly with the
+    Docker `env_file` configuration option (see
+    [this issue](https://github.com/moby/moby/issues/12997)).
+
+3.  Run `./derrick build` to build/re-build all the container images.
+
+4.  Run `./derrick up` to start the server(s).
+
+5.  Visit <http://localhost:8080/> in your browser.
+
+6.  When you\'re done working on Metecho, `Ctrl-C` in the terminal where the
+    containers are running to exit. You can also `./derrick down` to stop all
+    running containers, or `./derrick prune` to clean up unused
+    images/containers. (`docker-compose ps` will tell you what containers are
+    currently running.)
+
+## Setting up the GitHub App
+
+To deploy this app, you will need to set up a GitHub App and give it proper
+permissions. You can do that at
+`https://github.com/organizations/<your_org>/settings/apps`
+
+The GitHub app lets users log into Metecho with their GitHub account, connect to
+repositories, create branches, pull requests, and commit code. The app will need
+the following permissions:
+
+- Repository permissions
+  - Contents: Read & write
+  - Metadata: Read-only
+  - Pull requests: Read & write
+  - Commit statuses: Read & write
+  - Workflows: Read & write
+- Organization permissions:
+  - Members: Read-only
+- User permissions:
+  - Email addresses: Read-only
+- Subscribe to events:
+  - Pull request
+  - Pull request review
+  - Push
+
+If you want to allow Metecho to create new repositories in your organization,
+you must grant access to the app to all repositories, not a subset of them, and
+enable these additional permissions:
+
+- Repository permissions
+  - Administration: Read & write
+- Organization permissions
+  - Members: Read & write
+
+To enable logging in with GitHub, set the \"User authorization callback URL\" to
+`https://<your-deployed-url>/accounts/github/login/callback/`, and be sure the
+\"Request user authorization (OAuth) during installation\" box is checked.
```

Comment:
Do all quotations need to be escaped in markdown?

Reply:
I don't think so, seems to be a thing with the `pandoc` command that did the conversion

Reply:
Seems to work okay, just looks a bit busy.

Reply:
Fixed in bee068abcd35e2cbcb8e34a35a2221a55ec93c59